### PR TITLE
add font-feature-settings

### PR DESCRIPTION
**Pull request by SLemishko**
https://trello.com/c/9svkL6rn/25938-improve-quality-of-arabian-epubs-converted-in-a-fixed-layout

### CODE REVIEW

File: platforms/js/TextClip.hx [hunk 1]
```diff
@@ -2057,6 +2059,8 @@ class TextClip extends NativeWidgetClip {
 		TextClip.measureSVGTextElement.setAttribute('font-weight', computedStyle.fontWeight);
 		TextClip.measureSVGTextElement.setAttribute('letter-spacing', computedStyle.letterSpacing);
 		TextClip.measureSVGTextElement.setAttribute('white-space', computedStyle.whiteSpace);
+		var ffs = untyped __js__("computedStyle.fontFeatureSettings");
+		TextClip.measureSVGTextElement.style.fontFeatureSettings=ffs;
```
Comment (NikitaFil) [inline platforms/js/TextClip.hx:2063]: just TextClip.measureSVGTextElement.style.fontFeatureSettings = computedStyle.fontFeatureSettings

**Review by NikitaFil — APPROVED** [review]
LGTM

File: platforms/js/TextClip.hx (hunk 1 above)
Comment (AlexandrRzz) [inline platforms/js/TextClip.hx:2063]: Could you please add spaces around "="

**Review by AlexandrRzz — APPROVED** [review]
LGTM

**Pull request merged**